patternlab/styleguide/package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "@massds/mayflower",
   "description": "Open source UI components and visual style guide for Massachusetts government websites",
+  "engines": {
+    "node": ">6.9.0"
```

Comment:
Actually,  I just realized we can specify an `npm` engine version.   Let's make it `"npm": ">5.0.0"`, which is where they introduced packasge-lock.json.